### DISCUSSION TITLE
[Projects Beta] It's too hard to navigate back to the list of projects

### DISCUSSION THREAD

**Discussion by wojtekmaj** [community/community#6351] [Projects and Issues]
It's too hard to navigate back to the list of projects. Yes, you can click Back button, but if you accessed a Project using a bookmark for example, your only option is really to edit the URL. It shouldn't be this hard.

**Comment by stafyniaksacha** [comment]
Would be nice to have a sort of Project quick links (go back to org projects, go to XXX repository issues/discussions, etc...)

**Reply by tom-studiohakuna** [reply]
YES. Thank you. A Back arrow at least..

**Comment by chriskuech** [comment]
Seems to overlap with #9997
